scripts/migrations/olids_2026/verify_against_dbt_olids.py

```
@@ -0,0 +1,134 @@
+        # A staging column is "unknown" only if the SOURCE column doesn't exist
+        # in stable. The alias side is the downstream contract, so we don't care
+        # what staging chose to call it.
+        unknown = sorted(stg_source_set - stable_set)
+        unexposed = sorted(stable_set - stg_source_set)
+
+        if not (unknown or unexposed):
+            continue
+
+        print(f"\n=== {stg_sql.stem} vs {stable_stem} ===")
+        if unknown:
+            print(f"  UNKNOWN in upstream stable (staging refs columns that don't exist):")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)